templates/zerver/help/main.html

```diff
@@ -7,7 +7,20 @@
 <div class="app terms-page">
   <div class="app-main terms-page-container markdown">
   {{ article|render_markdown_path }}
+  <div id="footer">
+  <hr>
```

Comment:
I think this line ~ 19 , need to be indented by 2 spaces to improve readability of the code